redaxo/src/addons/structure/lib/perm.php

```diff
@@ -30,7 +30,7 @@ public function hasCategoryPerm($category_id)
      */
     public function hasStructurePerm()
     {
-        return $this->hasAll() || count($this->perms) > 0;
+        return !!$this->hasAll() || count($this->perms) > 0;
```

Comment:
Hmm, sollte das nicht vorher schon immer ein Boolean sein?
Aus `A || B` kommt in PHP doch immer ein Boolean raus.

Reply:
hmm mit javascript verwechelt... hmm